src/functions/module_filter.ts

```diff
@@ -66,6 +161,136 @@ export const FILTER = {
   isExported: true,
 } satisfies AddFunctionDescription;
 
+// -----------------------------------------------------------------------------
+// SORT
+// -----------------------------------------------------------------------------
+export const SORT: AddFunctionDescription = {
+  description: _t("Sorts the rows of a given array or range by the values in one or more columns."),
+  args: [
+    arg("range (range)", _t("The data to be sorted.")),
+    arg(
+      "sort_column (any, range<number>, repeating)",
+      _t(
+        "The index of the column in range or a range outside of range containing the values by which to sort."
+      )
+    ),
+    arg(
+      "is_ascending (boolean, repeating)",
+      _t(
+        "TRUE or FALSE indicating whether to sort sort_column in ascending order. FALSE sorts in descending order."
+      )
+    ),
+  ],
+  returns: ["RANGE"],
+  computeValueAndFormat: function (
+    range: Matrix<ValueAndFormat>,
+    ...datas: Arg[]
+  ): Matrix<ValueAndFormat> {
+    const _range = transposeMatrix(range);
+    return transposeMatrix(sortMatrix(_range, this.locale, ...datas));
+  },
+  isExported: true,
+};
+
+// -----------------------------------------------------------------------------
+// SORTN
+// -----------------------------------------------------------------------------
+export const SORTN: AddFunctionDescription = {
+  description: _t("Returns the first n items in a data set after performing a sort."),
+  args: [
+    arg("range (range)", _t("The data to be sorted.")),
+    arg("n (number, default=1)", _t("The number of items to return.")),
+    arg(
+      "display_ties_mode (number, default=0)",
+      _t("A number representing the way to display ties.")
+    ),
+    arg(
+      "sort_column (number, range<number>, repeating)",
+      _t(
+        "The index of the column in range or a range outside of range containing the values by which to sort."
+      )
+    ),
+    arg(
+      "is_ascending (boolean, repeating)",
+      _t(
+        "TRUE or FALSE indicating whether to sort sort_column in ascending order. FALSE sorts in descending order."
+      )
+    ),
+  ],
+  returns: ["RANGE"],
+  computeValueAndFormat: function (
+    range: Matrix<ValueAndFormat>,
+    n: Maybe<ValueAndFormat>,
+    displayTiesMode: Maybe<ValueAndFormat>,
+    ...values: (ValueAndFormat | Matrix<ValueAndFormat>)[]
+  ): any {
+    const _n = toNumber(n?.value ?? 1, this.locale);
+    assert(
+      () => _n >= 0,
+      _t(`Wrong value of 'n'. Expected a positive number. Got %s.`, toString(_n))
+    );
+    const _displayTiesMode = toNumber(displayTiesMode?.value ?? 0, this.locale);
+    assert(
+      () => _displayTiesMode >= 0 && _displayTiesMode <= 3,
+      _t(
+        `Wrong value of 'display_ties_mode'. Expected a positive number between 0 and 3. Got %s.`,
+        toString(_displayTiesMode)
+      )
+    );
+    const sortedData = sortMatrix(transposeMatrix(range), this.locale, ...values.slice(2));
```

Comment:
isn't `slice(2)` a leftover of a previous version ? Is it working ? (maybe there's a missing test ^^)